webpack.config.js

```diff
@@ -158,7 +158,10 @@ function getWebpackConfig( {
 	const webpackConfig = {
 		bail: ! isDevelopment,
 		context: __dirname,
-		entry: { build: [ path.join( __dirname, 'client', 'boot', 'app' ) ] },
+		entry: {
```

Comment:
Adding a new entrypoint seems like a pretty fundamental change to how things are currently being done.

@blowery: Does this approach work well with any potential future plans for Calypso around splitting things up into smaller builds?

Reply:
I don't really know, since we don't currently have any solid plans. I don't see it hurting anything though and could open some doors. Each entry point can have its own boot and setup, which may make things easier to isolate.

An enhancement of what we have so far might be to break out a manifest per entry point instead of our current setup with one manifest.

Reply:
Is this something you think we should address before continuing? Or something we should address in a future PR?